static/styles/portico/portico_signin.css

```diff
@@ -698,7 +698,7 @@ html {
 button.login-social-button {
     width: 328px;
     height: auto;
-    padding-left: 35px;
+    padding-left: 60px;
```

Comment:
While this works, this doesn't seem like the sort of clean fix that would avoid future regressions.

Or maybe it is -- how did you pick 60px?  Is there math on the size of elements we should be using instead?

Reply:
Yes, I did calculate for 'Sign up with Google' button, the background-position is 13px, and the logo's height is 60% of the button, i.e. 50px * 0.6 = 30px. Thus, the width of the logo is (30/76)*74 = 29px approximately. Now, the '.new-style button' had a 22px padding on the right of the text. So, to center the text between the right edge of the logo and the right edge of the button and to have an equal space to the left of the text (i.e. 22px) , the padding should be = 13px + 29px +  22px = 64px. I visually adjusted it to 60px as other logos had slightly different aspect ratios.

Reply:
Hey @timabbott @alya I would love to address any other concerns you have. Please review so that I can work on new issues.. Thanks!